app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,5 @@
+class VendorsController < ApplicationController
+  def new
+  binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Use 2 (not 0) spaces for indentation.